lang/en.lyaml

```diff
@@ -1202,6 +1202,7 @@ en:
         selectReactType: "Is this a React module?"
         selectContentType: "What types of content will this module be used in?"
         confirmGlobal: "Is this a global module?"
+        availableForNewContent: "Is this available for new content?"
```

Comment:
Would just adjust the wording here to `Make this module available for new content?`